### PR TITLE
Feat: Show wild Loomies in 3D map

### PR DESCRIPTION
**Pull request by Woynert**
# What's new

- Fetches Loomies near the player periodically.
- Loomies get stored and only those who change (aka got deleted or just spawned) are updated.
- Loomies are repositioned when the player is out of bounds.
- Fixes the methods for fetching gyms and Loomies, so it tries to refresh the Access-Token before giving up.
- Optimized the way the models are created. This optimization caused some models to have some body parts rotated in the wrong direction but nothing hard to fix.
- Fixes some models issues like de Spiky Crab (005) having a twisted leg. There could be other models with these problems but we'll fix them as they are reported.

image

- It also adds the Loomie 019 which was missing from #45.

image

# How to try

Hop onto the app and you should be able to see some Loomies lurking around.

You can preview the models using this tool (https://sandbox.babylonjs.com/).

### CODE REVIEW

**Review by PedroChaparro — APPROVED** [review]
Good work. I got around 20-25 FPS using the DEBUG mode.

File: src/services/map.services.ts [hunk 1]
```diff
@@ -54,6 +61,68 @@ export const getNearGyms = async (
...
+  try {
+    // get access token from the storage
+    const [accessToken, error] = await getStorageData('accessToken');
+    if (error || !accessToken) {
+      return null;
+    }
+
+    // make request
+    const response = await Axios.post(
+      `${API_URL}/near_loomies`,
```
Comment (PedroChaparro) [inline src/services/map.services.ts:93]: This endpoint is `/loomies/near` now.

**Review by PedroChaparro — CHANGES_REQUESTED** [review]
You should update the branch to use the up to date endpoints.

**Review by PedroChaparro — APPROVED** [review]

**Comment by SilviaPabon** [comment]
I obtained between 19 and 47 fps, but as general, values between 20 and 30 fps

File: src/utils/utils.ts [hunk 2]
```diff
@@ -35,5 +35,6 @@ export const images: {
   '015': require('@assets/images/loomies/015.png'),
   '016': require('@assets/images/loomies/016.png'),
   '017': require('@assets/images/loomies/017.png'),
-  '018': require('@assets/images/loomies/018.png')
+  '018': require('@assets/images/loomies/018.png'),
+  '019': require('@assets/images/loomies/018.png')
 };
```
Comment (SilviaPabon) [inline src/utils/utils.ts:40]: Houston 🎙️ @Woynert Are you sure that '019' is the 0.18.png?

Comment (SilviaPabon) [reply src/utils/utils.ts:40]: @Woynert 📢 🚨

**Review by SilviaPabon — APPROVED** [review]

**Pull request merged**